src/Npgsql/PostgresDatabaseInfo.cs

```diff
@@ -116,6 +121,7 @@ internal async Task LoadPostgresInfo([NotNull] NpgsqlConnection conn, NpgsqlTime
         /// <param name="withEnum">True to load enum types.</param>
         /// <param name="withEnumSortOrder"></param>
         /// <param name="loadTableComposites">True to load table composites.</param>
+        /// <param name="withTypCategory">Field pg_type.typcategory is added after 8.4. refer: https://www.postgresql.org/docs/8.4/static/catalog-pg-type.html#CATALOG-TYPCATEGORY-TABLE</param>
```

Comment:
Could you move this comment up to `HasTypeCategory` as a `<remarks>` after the `<summary>` block? Then give this a shorter description? (Eases the intellisense popup.)

E.g.
```xml
/// <summary>
/// True if the 'pg_type' table includes the 'typcategory' column; otherwise, false.
/// </summary>
/// <remarks>
/// pg_type.typcategory is added after 8.4. 
/// see: https://www.postgresql.org/docs/8.4/static/catalog-pg-type.html#CATALOG-TYPCATEGORY-TABLE
/// </remarks>
```

Reply:
Also change this param to `withTypeCategory`.